src/internal/connector/data_collections_test.go

```diff
@@ -180,24 +181,27 @@ func (suite *ConnectorDataCollectionIntegrationSuite) TestSharePointDataCollecti
 
 	for _, test := range tests {
 		suite.T().Run(test.name, func(t *testing.T) {
-			_, err := connector.SharePointDataCollections(ctx, test.getSelector(t))
+			collection, err := sharepoint.DataCollections(
+				ctx,
+				test.getSelector(t),
+				connector.GetSiteIds(),
+				connector.credentials.AzureTenantID,
+				connector)
 			require.NoError(t, err)
+			assert.Equal(t, 1, len(collection))
```

Comment:
should this check that it's equal to the length of the number of site IDs? Or are we getting only a single collection for all sites?
```suggestion
			assert.Len(t, collection, len(connector.GetSiteIds())
```

Reply:
I think it's mocked, but this is a good change nonetheless.